packages/gatsby-theme-code-recipes/src/recipe.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { RecipeQuery } from '../types'
+
+type Recipe = Omit<RecipeQuery, 'body'> & {
+  children: React.ReactNode
+}
+
+/** Props such as a `name` etc. are passed down and used by other packages */
```

Comment:
This can also be shadowed, right?

Reply:
Looks like it is already shadowed by other package `packages/docs/src/gatsby-theme-code-recipes/recipe.js`. Anyway I will remove this comment because it doesn't do its job (TBH have no clue what I was trying to tell here).